src/graphics/canvas.rs

```diff
@@ -590,3 +590,8 @@ pub fn default_text_shader() -> Shader {
         vs_module: None,
     }
 }
+
+/// The default sampler
+pub fn default_sampler() -> Sampler {
+    Sampler::linear_clamp()
+}
```

Comment:
That part we should definitely keep though.

Reply:
Ok, solved in https://github.com/ggez/ggez/pull/1121 by implementing `Default` for `Sampler`.